packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -327,24 +327,35 @@ export const ReleaseDetailsLayout = ({
   );
 };
 
-const GROUP_BY_OPTIONS = [
-  {
-    label: 'Content Type',
-    value: 'contentType',
-  },
-  {
-    label: 'Locale',
-    value: 'locale',
-  },
-  {
-    label: 'Action',
-    value: 'action',
-  },
-];
-
 /* -------------------------------------------------------------------------------------------------
  * ReleaseDetailsBody
  * -----------------------------------------------------------------------------------------------*/
+const GROUP_BY_OPTIONS = ['contentType', 'locale', 'action'] as const;
+const getGroupByOptionLabel = (value: (typeof GROUP_BY_OPTIONS)[number]) => {
+  switch (value) {
+    case 'contentType':
+      return {
+        id: 'content-releases.pages.ReleaseDetails.groupBy.option.content-type',
+        defaultMessage: 'Content-Types',
+      };
```

Comment:
maybe we can reduce a little bit the code using the default for this case and remove this part